.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: CI bot etu UTT
+
+
+on:
+  pull_request:
+    branches:
+      - main
+
+
+jobs:
+  ruff-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: chartboost/ruff-action@v1
+
+  ruff-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: chartboost/ruff-action@v1
+        with:
+          args: "format . --diff"
```

```suggestion
          args: "format --diff"
```
le point ne sert à rien c'est le comportement par défaut